src/sql/src/session/vars.rs

```diff
@@ -959,6 +959,15 @@ const KEEP_N_SOURCE_STATUS_HISTORY_ENTRIES: ServerVar<usize> = ServerVar {
     internal: true
 };
 
+/// Whether compute rendering should use Materialize's custom linear join implementation rather
+/// than the one from Differential Dataflow.
+const STORAGE_CLIENT_FINALIZE_SHARDS: ServerVar<bool> = ServerVar {
+    name: UncasedStr::new("storage_client_finalize_shards"),
```

Comment:
I think we normally name these boolean vars with an `enable_` prefix?

```suggestion
    name: UncasedStr::new("enable_storage_shard_finalization"),
```